content/10.diagnostics.md

```diff
@@ -3,7 +3,7 @@
 ### Abstract
 
 The COVID-19 pandemic has presented a number of challenges that have spurred biotechnological research aiming to address specific problems.
-One important area of biotechnology research is diagnostics.
+Diagnostics is one area where biotechnology has been particularly important.<!--Alt: One important area of biotechnology research is diagnostics.-->
```

Comment:
This line was rewritten here and in #1120. I will default to the version that is not in the comments unless there is a preference for the other!